spec/driverProviderTest.js

```diff
@@ -0,0 +1,138 @@
+/**
```

Comment:
Oh, neat!

Reply:
I enabled this test (it wasn't running in the test suite and wasn't part of test.js). I also renamed it.